waffle/jinja.py

```diff
@@ -15,9 +15,11 @@ def inline_wafflejs_helper(context):
     return _generate_waffle_js(context['request'])
 
 
-jingo.env.globals['waffle'] = {
-    'flag': flag_helper,
-    'switch': switch_is_active,
-    'sample': sample_is_active,
-    'wafflejs': inline_wafflejs_helper
-}
+class WaffleExtension(Extension):
```

Comment:
I stole most of your code from #167, so the 1.8 tests actually do run with django-jinja! :)